ovh/client.py

```diff
@@ -484,3 +449,61 @@ def call(self, method, path, data=None, need_auth=True):
             raise NetworkError()
         else:
             raise APIError(json_result.get('message'), response=result)
+
+    def raw_call(self, method, path, data=None, need_auth=True):
+        """
+        Lowest level call helper. If ``consumer_key`` is not ``None``, inject
+        authentication headers and sign the request.
+        Will return a vendored ``requests.Response`` object.
+
+        Request signature is a sha1 hash on following fields, joined by '+'
+         - application_secret
+         - consumer_key
+         - METHOD
+         - full request url
+         - body
+         - server current time (takes time delta into account)
+
+        :param str method: HTTP verb. Usually one of GET, POST, PUT, DELETE
+        :param str path: api entrypoint to call, relative to endpoint base path
+        :param data: any json serializable data to send as request's body
+        :param boolean need_auth: if False, bypass signature
+        :raises HTTPError: when underlying request failed for network reason
+        :raises InvalidResponse: when API response could not be decoded
```

Comment:
If I understand the code fully, it does not raise these but may raise ``RequestException`` which is OK as this call is branded as "raw request". Can you update the doc ?